pyppeteer/page.py

```diff
@@ -1515,8 +1535,13 @@ def mouse(self) -> Mouse:
         """Get :class:`~pyppeteer.input.Mouse` object."""
         return self._mouse
 
-    async def click(self, selector: str, options: dict = None, **kwargs: Any
-                    ) -> None:
+    async def click(
+            self,
+            selector: str,
+            delay: float = None,
+            button: str = None,  # left, right, middle
```

Comment:
adding marker for #30 
```suggestion 
            # todo: better typing
            button: str = None,  # left, right, middle
```